### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by pabs3**
Also fix some confusion with American vs International English spelling.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -7830,12 +7841,15 @@ intiailizing->initializing
 intial->initial
 intialisation->initialisation
 intialise->initialise
-intialised->initialized
-intialises->initializes
+intialised->initialised
+intialiser->initialiser
+intialises->initialises
+intialises->initialises
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: From Travis

u'intialises' already exists

**Comment by pabs3** [comment]
> u'intialises' already exists

Yes but it is broken, it changes the misspelled International English
word into a correctly spelled but American English word. Same for the
other updates of similar words.

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: That's not sure the problem -- it's complaining that there is now a duplicate entry. It's in there twice if you look at the diff (here).

Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: Lines 7846 and 7847 are identical in this case, delete one and you should be good

**Comment by pabs3** [comment]
Woops forgot to run `make sort-dictionary` after fixing the commit.

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Comment by pabs3** [comment]
Sorting the dictionary also removes duplicates.

Anyway, updated the commit.

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Comment by larsoner** [comment]
Now Travis says

AssertionError: error diabler: correction disabler is an error itself

So if disabler is to be taken as valid, the correction for it should be removed

**Comment by pabs3** [comment]
I found the diabler misspelling in the wild and it seems to indeed be a
misspelling for disabler not disabled. In addition disabler seems to be
a valid word so I'm inclined to drop the disabler->disabled check.
Thoughts?

https://codesearch.debian.net/search?q=diabler+package:gforth
https://www.thefreedictionary.com/disabler

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Comment by larsoner** [comment]
Yeah I'm okay with removing `disabler` as an error

**Pull request merged**